setup.py

```diff
@@ -31,5 +30,5 @@
                    'firedrake_adjoint': 'firedrake_adjoint', 'dolfin_adjoint': 'dolfin_adjoint',
                    'dolfin_adjoint_common': 'dolfin_adjoint_common', 'numpy_adjoint': 'numpy_adjoint'},
       install_requires=['scipy>=1.0'],
-      extras_require=extras,
-      )
+      extras_require=extras
+)
```

Comment:
Leave the indent as before to match visual indent :)
```suggestion
      )
```